click_tester/click_runner_test_case.py

```diff
@@ -36,6 +37,12 @@ def expect_invoke_raises_error(self, cmd, opts, exception_class=SystemExit,
         self.assertIsInstance(result.exception, exception_class)
 
         if expected_output_regexp is not None:
-            self.assertRegexpMatches(result.output, expected_output_regexp)
+            self.checkRegex(result.output, expected_output_regexp)
 
         return result
+
+    def checkRegex(self, text, expected_regex):
+        if sys.version_info < (3, 0):
+            return self.assertRegexpMatches(text, expected_regex)
+
+        return self.assertRegex(text, expected_regex)
```

Comment:
@michellejzh This could also be accomplished by overriding `assertRegexp` and then conditionally calling the super class method.  That would avoid introducing a 3rd method into the mix.